src/components/Onboarding/Screens/HoneyswapLiquidity.js

```diff
@@ -92,21 +106,33 @@ function HoneyswapLiquidity() {
         : denominatedAmount
     ) >= 100
 
-  const nextEnabled = liquidityProvided && satisfiesMin
-
   const tokenSymbol =
     config.garden.type === BYOT_TYPE
       ? config.tokens.existingTokenSymbol
       : config.tokens.symbol
 
-  const convertedValue = parseFloat(
-    denom === HNY_DENOMINATION
-      ? denominatedAmount * hnyPrice
-      : denominatedAmount / hnyPrice
-  ).toFixed(2)
+  const equivalentValue = String(
+    parseFloat(
+      denom === HNY_DENOMINATION
+        ? denominatedAmount * hnyPrice
+        : denominatedAmount / hnyPrice
+    ).toFixed(2)
+  )
 
   const hnyAmount =
-    denom === HNY_DENOMINATION ? denominatedAmount : convertedValue
+    denom === HNY_DENOMINATION ? denominatedAmount : equivalentValue
+
+  const hasSufficientHNYBalance = bigNum(hnyAmount, 18).lte(hnyTokenBalance)
+  const hasSufficientETokenBalance = bigNum(
```

Comment:
i think that we have typo on the `E` right should't be `hasSufficientTokenBalance` or does it mean something else?